crates/api_common/src/site.rs

```diff
@@ -252,6 +253,8 @@ pub struct EditSite {
   pub federation_debug: Option<bool>,
   /// The number of federation workers.
   pub federation_worker_count: Option<i32>,
+  /// The number of federation retry workers.
+  pub federation_retry_worker_count: Option<i32>,
```

Comment:
I missed that your PR was adding this. In fact I want to get rid of the worker count setting, not add another one...

Reply:
I've raised a PR to have this as an option: https://github.com/LemmyNet/activitypub-federation-rust/pull/45 if you set both these values to `0` then there isn't any limit.